### ISSUE TITLE
BUG: mmpm command not found

### ISSUE THREAD

**Issue by IdoPrinc3** [Bee-Mar/mmpm#134]
**Describe the Bug**

mmpm: command not found
when trying to install mmpm

Is this an issue with the CLI or GUI?
**CLI**

Is this something that should be put in the Wiki?
if it's a common issue/fix then yes

New installation of Raspberry Pi OS (more info at the end of the page)
Installed magicMirror successfully and it works (0 modules) 

now following mmpm installation guide
1. "sudo apt install libffi-dev nginx-full -y" - works (output is at the end of the issue)
2. "python3 -m pip install --upgrade --no-cache-dir mmpm" - works (output is at the end of the issue)
3. "mmpm --guided-setup" or any "mmpm" command -

"bash: mmpm: command not found"

**To Reproduce**
follow the quick installation guide, type mmpm and see the error

**Expected behavior**
begin to use the mmpm command successfully.

**Operating System, Python3 Version, and Web Browser**\

Raspberry Pi OS with desktop (February 21st 2023)
Debian version: 11 (bullseye)

- output from `cat /etc/os-release`
PRETTY NAME= "Raspbian GNU/Linux 11 (bullseye)"
NAME= "Raspbian GNU/Linux"
(VERSION ID="11"
VERSION="11 (bullseye)"
VERSION_ CODENAME=bullseye
ID=raspbian
ID LIKE=debian
HOME_URL-"http://www.raspbian.org/"
SUPPORT_URL="http: //www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

- output from `python3 --version`
- Python 3.9.2

- Web Browser (firefox, chrome, safari, chromium, etc)
- chromium

**MMPM Environment Settings**

- output from `mmpm env`
- "bash: mmpm: commant not found"

**MMPM Log Files**

- Attach the file produced `Download MMPM log files` within the Control Center of the GUI or from the  `mmpm log --zip`
- "bash: mmpm: commant not found"

**MagicMirror Config**

- your MagicMirror `config.js`

{
    "compilerOptions":
     {
        "target": "es6"
        "module": "commonjs"
        "allowSyntheticDefaultImports": true
    },
    "exclude": ['modules", "node modules"]
}

**Comment by Bee-Mar** [comment]
@IdoPrinc3 did you add $HOME/.local/bin to your PATH as indicated on the GitHub repo's front page?

**Comment by IdoPrinc3** [comment]
@Bee-Mar
firstly, I want to state that I'm familiar with code but I don't operate in a linux environment and this is my first time using RPi

Because I got stuck on the "mmpm --guided-setup" I never  tried the commands that come after it
Also the readme file and the wiki installation instruction are different which can be confusing

allow me to suggest some changes -

1. "mmpm --guided-setup" -
the command to add to PATH should come before this command.

2. "echo 'export PATH="$PATH:$HOME/.local/bin"' >> ~/.bashrc" -
why is it written like that and not just: **export PATH="$PATH:$HOME/.local/bin"** so people could copy-paste it.

After I added to path the mmpm command did work and I installed it.

I had small difficulties when installing
the question: "Did you install MMPM as a Docket image, or using docker-compose?"
well as I'm less familiar with docker I had to look it up online to find no concrete answer but my intuition is that i'm using a docker image.
It was difficult for me because I just installed it using bash commands, not knowing what it was doing behind the scenes

also just a small typo
when it's asking if I use the localhost address to open magicMirror the address that it is asking me about is "127.0.1.1"
the 3rd number should be a 0 instead of 1. 

If you want I can open a new issue about the localhost typo I just wanted to make sure with you that it is indeed a typo.

I thank you for your help and dedication
